Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/Concepts.java

```diff
@@ -47,9 +55,40 @@
 
     private static final Logger logger = LoggerFactory.getLogger(Concepts.class);
 
+    /**
+     * This method creates concepts as well as concept wrappers. This is decided
+     * based on the request parameter. If the request parameter contains the
+     * wrapper ids then concept is wrapped else the new concept is created.
```

Comment:
And also "then a new concept wrapper is created, otherwise a new concept is created."